src/System.Runtime.Serialization.Formatters/tests/EqualityExtensions.cs

```diff
@@ -903,6 +903,20 @@ public static bool IsEqual(this Version @this, Version other)
                 @this.MinorRevision == other.MinorRevision;
         }
 
+        public static bool IsEqual(this Exception @this, Exception other)
+        {
+            return @this != null &&
+                other != null &&
+                // On full framework, line number may be method body start
+                (PlatformDetection.IsFullFramework ? true :
+                (@this.StackTrace == other.StackTrace &&
+                @this.ToString() == other.ToString())) &&
+                BinaryFormatterTests.CheckSequenceEquals(@this.Data, other.Data) &&
+                @this.Message == other.Message &&
+                @this.Source == other.Source &&
+                @this.HResult == other.HResult;
```

Comment:
These are all the properties that @stephentoub checked on Exception types before. Should we add more?